src/test/java/nextstep/favorite/acceptance/FavoriteAcceptanceTest.java

```diff
@@ -1,8 +1,220 @@
+    }
+
+    /**
+     * Given 회원 가입한 사용자가 있고,
+     * And 해당 사용자가 로그인을 한 상태이고,
+     * And 해당 사용자가 즐겨찾기를 등록한 상태일 때
+     * When 해당 사용자의 즐겨찾기를 조회 하면
+     * Then 해당 사용자가 등록한 즐겨찾기가 조회된다.
+     */
+    @Test
+    @DisplayName("사용자는 자신이 등록한 즐겨찾기를 조회할 수 있다.")
+    void getFavorites() {
+        // given
+        String accessToken = 사용자_설정_및_로그인(EMAIL, PASSWORD);
+
+        // 즐겨찾기 생성
+        즐겨찾기_생성_요청(accessToken, 강남역_ID, 신사역_ID);
+
+        // when
+        ExtractableResponse<Response> 즐겨찾기_조회_응답 = 즐겨찾기_조회(accessToken);
+
+        // then
+        assertThat(즐겨찾기_조회_응답.statusCode()).isEqualTo(HttpStatus.OK.value());
+
+        List<Map<String, Object>> favorites = 즐겨찾기_조회_응답.jsonPath().getList(".");
+        assertThat(favorites).hasSize(1);
+
+        Map<String, Object> favorite = favorites.get(0);
+        assertThat(favorite).containsKeys("id", "source", "target");
+
+        Map<String, Object> source = (Map<String, Object>) favorite.get("source");
+        Map<String, Object> target = (Map<String, Object>) favorite.get("target");
+
+        assertThat(source.get("id")).isEqualTo(강남역_ID.intValue());
+        assertThat(source.get("name")).isEqualTo("강남역");
+        assertThat(target.get("id")).isEqualTo(신사역_ID.intValue());
+        assertThat(target.get("name")).isEqualTo("신사역");
+    }
+
+    /**
+     * Given 회원 가입한 사용자가 있고,
+     * And 해당 사용자가 로그인을 한 상태이고,
+     * And 해당 사용자가 즐겨찾기를 등록한 상태일 때
+     * When 해당 사용자가 즐겨찾기를 삭제 하면
+     * Then 헤딩 사용자가 등록한 즐겨찾기가 삭제된다.
+     */
+    @Test
+    @DisplayName("사용자는 자신이 등록한 즐겨찾기를 삭제할 수 있다.")
+    void deleteFavorite() {
+        // given
+        String accessToken = 사용자_설정_및_로그인(EMAIL, PASSWORD);
+        Long favoriteId = 즐겨찾기_생성_및_ID_추출(accessToken, 강남역_ID, 신사역_ID);
+
+        // when
+        ExtractableResponse<Response> 즐겨찾기_삭제_응답 = 즐겨찾기_삭제_요청(accessToken, favoriteId);
+
+        // then
+        assertThat(즐겨찾기_삭제_응답.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+
+        ExtractableResponse<Response> 즐겨찾기_조회_응답 = 즐겨찾기_조회(accessToken);
+        List<Map<String, Object>> favorites = 즐겨찾기_조회_응답.jsonPath().getList(".");
+        assertThat(favorites).isEmpty();
+    }
+
+    /**
+     * Given 회원 가입한 사용자가 있고,
+     * And 헤딩 사용자가 로그인을 한 상태이고,
+     * And 해당 사용자가 즐겨찾기를 등록한 상태일 때
+     * When 다른 사용자가 해당 사용자의 즐겨찾기를 삭제 하면
+     * Then 즐겨찾기가 삭제가 실패하고 권한 오류가 발생한다.
+     */
+    @Test
+    @DisplayName("다른 사용자의 즐겨찾기를 삭제하려고 하면 인증 오류가 발생한다.")
+    void deleteOtherUsersFavorite() {
+        // given
+        String accessToken = 사용자_설정_및_로그인(EMAIL, PASSWORD);
+        Long favoriteId = 즐겨찾기_생성_및_ID_추출(accessToken, 강남역_ID, 신사역_ID);
+        String otherAccessToken = 사용자_설정_및_로그인("other@email.com", "otherpassword");
+
+        // when
+        ExtractableResponse<Response> 즐겨찾기_삭제_응답 = 즐겨찾기_삭제_요청(otherAccessToken, favoriteId);
+
+        // then
+        assertThat(즐겨찾기_삭제_응답.statusCode()).isEqualTo(HttpStatus.FORBIDDEN.value());
```

Comment:
```suggestion
        var accessToken = 사용자_설정_및_로그인(EMAIL, PASSWORD);
        var favoriteId = 즐겨찾기_생성_및_ID_추출(accessToken, 강남역_ID, 신사역_ID);
        var otherAccessToken = 사용자_설정_및_로그인("other@email.com", "otherpassword");

        // when
        var 즐겨찾기_삭제_응답 = 즐겨찾기_삭제_요청(otherAccessToken, favoriteId);

        // then
        assertThat(즐겨찾기_삭제_응답.statusCode()).isEqualTo(HttpStatus.FORBIDDEN.value());
```

권한 오류 테스트 👍👍 

추가로, 타입 추론을 사용해 볼까요? 타입 추론을 사용하면 간결해지고 직관적인 효과가 있어요. 추가로 변수명도 의미있게 지을 수도 있더라구요.

Reply:
타입 추론을 잘 사용하지 않게 되는 이유가, 반복적으로 타입을 확인하러 메소드 시그니쳐를 확인하는 것이 번거롭다고 느껴서 인데요. 변수명이 의미있게 지어진다는 것은, 타입에 국한 하지 않고 맥락에 맞게 지을 수 있다는 의미 이실까요? 일단 타입 추론 자체를 일하면서 별로 안써보았으니 미션에서는 역으로 더 사용을 해보도록 하겠습니다!

Reply:
네네 맞아요. 맥락에 맞게 지을 수 있다는 것이죠. 타입 추론에 대해서는 많은 언어들도 지원을 해주고, 코틀린인 경우에는 타입추론이 기본적이기도해요.